packages/terra-app-delegate/docs/README.md

```diff
@@ -145,3 +134,7 @@ ExampleRootComponent.propTypes = {
 }
 
 ```
+
+### Deprecated APIs
+
+The Disclose API for specifying which component to present was previously structured to support Redux-based disclosure managers. However, the disclosure managers were transitioned away from Redux to allow consumers to have greater control over the disclosure lifecycle. Therefore, the `name` and `props` keys in the disclose API are now deprecated, and the `component` key should be used to provide a component instance to present instead. This also means that the `registerComponentForDisclosure`/`getComponentForDisclosure` are deprecated as well, as there is no need for component retreival when an instance is given through the API. Those APIs will continue to function, but the usage of the `component` key is recommended.
```

Comment:
> Therefore, the `name` and `props` keys in the disclose API are now deprecated, and the `component` key should be used to provide a component instance to present instead.

This this be an enhancement we will need to make to terra-site?

Reply:
If terra-site is using the deprecated APIs, things will still work for the time being. I'm not sure what our time frames would be for removing the deprecated APIs, but it will be at least after the React 16 bump.

Reply:
Yeah let's try and keep the bump to react 16 as passive as possible. Post react 16 release, we can remove the deprecated APIs